File: SkinHuntApp/SkinHunt.Application/Common/Models/SkinModel.cs
namespace SkinHunt.Application.Common.Models
{
    public class SkinModel
    {
        public string Name { get; set; }

        public ItemType Type { get; set; }

        public DateTime ReleaseDate { get; set; }

        public double Float { get; set; }

        public string Color { get; set; }

        public string Rarity { get; set; }

        public decimal Price { get; set; }

        public bool IsDiscount { get; set; }

        public decimal PriceWithDiscount { get; set; }
    }
}

File: SkinHuntFront/src/app/models/skinModel.ts
export interface SkinModel {
    id: string;
    name: string;
    type: {
        category: number;
        id: string;
        subcategory: number;
    };
    color: string;
    float: number;
    price: number;
    isDiscount: boolean;
    priceWithDiscount: number;
    rarity: string;
    releaseDate: string;
    photo: string;
  }
  